src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,23 @@
+.content__catalog {
+  margin: 0 auto;
+}
+
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: (200px * 4) + (48px * 3);
```

Comment:
not a recommendation, just my opinion: I think it's better not to use calculations in width

Reply:
Use function for it, and create some variables